doc/source/api/visualization/graphics_windows_manager.rst

```diff
@@ -6,40 +6,39 @@ The ``GraphicsWindowsManager`` class provides for managing and directly interact
 with PyVista windows. By registering these methods with ``EventsManager``, you
 can update graphics during run time and create animations.
 
-The following example updates ``window-1`` during solution initialization and
-whenever data is read. During the calculation, it also updates ``window-1`` at
+The following example updates ``p_cont`` during solution initialization and
+whenever data is read. During the calculation, it also updates ``p_cont`` at
 the end of every time step and creates an animation.
 
 .. note::
   The animation is saved when the window is closed.
 
 .. code-block:: python
 
-    from ansys.fluent.visualization import Graphics
-    from ansys.fluent.visualization.graphics import graphics_windows_manager
-    
-    graphics_session = Graphics(session)
+    from ansys.fluent.visualization import Contour, GraphicsWindow
     
     #Create contour.
-    contour1 = graphics_session.Contours["contour-1"]
-    contour1.field = "velocity-magnitude"
-    contour1.surfaces = ['symmetry']
+    contour1 = Contour(
+        solver=session, field="velocity-magnitude", surfaces=["symmetry"]
+    )
     
-    #Display contour on window-1.
-    contour1.display("window-1")
+    #Display contour.
```

Comment:
@prmukherj  If we do retain this comment then it should be moved to where the contour is shown. However, once everything is naming appropriately, there seems so no need for the comment.